Articles/Blog/HelloHeatEquation.md

```diff
@@ -0,0 +1,226 @@
+# The Heat Equation is the "Hello World" of Scientific Computing
+
+#### Contributed by [Heather M. Switzer](https://github.com/heatherms27 "Heather Switzer GitHub Profile") and [Elsa Gonsiorowski](https://github.com/gonsie "Elsa Gonsiorowski GitHub Profile")
+
+"Hello World!" is a prototypical program that is universally used by programmers to illustrate the basic syntax of a programming language. At a glance, it can demonstrate a programming language's syntactic structure and can indicate how easy or difficult implementing a basic program may be (see several [Hello World implementations at WikiBooks](https://en.wikibooks.org/wiki/Computer_Programming/Hello_world)). For the scientific software community, a programming language is only the most basic building block of a program. Scientific software must use various programming models or math libraries to achieve high performance for any particular problem. These communities need a version "Hello World!" to compare programming options. The Heat Equation, a well known partial differential equation (PDE), is the perfect formula for this task.
+
+## What is the Heat Equation?
+The Heat Equation, also known as the Diffusion Equation, was developed by Joseph Fourier in 1822. Primarily used in the fields of Mathematics and Physics, the equation shows how heat (or something similar) changes over time in a solid medium. While there are many adaptations for different dimensions and scenerios, new programmers are often first introduced to the 1-dimensional version of the Heat Equation as a tutorial problem. Specifically, the Forward-Time, Central-Space (FTCS) version of the equation is used.
+
```

Comment:
`FTCS` really only refers to the discretization phase of the algorithm implementation, not the theoretical exposition of the science problem itself.

Reply:
Good catch. @heatherms27 will review the text to fix-up the wording around the algorithm.